clash-bitpackc/src/Clash/Class/BitPackC.hs

```diff
@@ -29,6 +29,13 @@ import Data.Proxy (Proxy (Proxy))
 import Data.Word (Word16, Word32, Word64, Word8)
 import GHC.Generics
 
+{- | Maximum alignment boundary of any type. This is currently hardcoded to 8,
+which is what our platforms use (riscv32). In the future this should be made
+configurable to support platforms with larger or smaller maximum alignment
+requirements.
+-}
```

Comment:
Link to the issue? (it doesn't exist yet, but I thought I'd leave a comment here

Reply:
Done!